docs/pages/machine-workload-identity/mwi-use-cases/mwi-use-cases.mdx

```diff
@@ -0,0 +1,43 @@
+---
+title: Machine & Workload Identity Use Cases
+description: Common use cases with Machine ID & Workload Identity
+labels:
+ - conceptual
+ - mwi
+---
+
+Machine & Workload Identity can work together to create a comprehensive security model. Machines can securely access resources while workloads communicate securely with each other and external services, 
+all managed through Teleport's unified access plane.
+
+## CI/CD pipeline with end-to-end authentication
+
+A [CI/CD system](../mwi-use-cases/mwi-ci-cd.mdx) securely deploys services to Kubernetes and establishes secure communication channels between them:
+
+- The pipeline authenticates through the [Teleport Proxy Service](../../reference/architecture/architecture.mdx#teleport-proxy-service) to deploy to Kubernetes and receives credentials to interact with cloud APIs (e.g., to push container images)
+- Services deployed by the pipeline receive SPIFFE identities for mutual TLS. The pipeline manages the identity lifecycle for the services it deploys
+
+## Cloud-native application with third-party API access
+
+A Kubernetes-based application needs access to both internal services and external APIs:
+
+- Automation tools authenticate to configure the cluster securely
+- Application components are issued SPIFFE identities
+- Identities authenticate to internal services via mTLS
+- JWT-based authentication is used for external API access
```

Comment:
Something about this section feels off to me. I think it might be too abstract. It feels less like a specific use case and more like mashup of a bunch of different technologies.